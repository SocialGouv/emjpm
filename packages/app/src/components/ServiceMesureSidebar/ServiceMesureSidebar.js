import { ArrowBack } from "@styled-icons/boxicons-regular/ArrowBack";

import { Box, Flex, Link as StyledLink, Text } from "rebass";

import { Link } from "~/components/Link";

const linkStyle = { color: "black", fontSize: "1", my: "3" };

function ServiceMesureSidebar(props) {
  return (
    <Box {...props}>
      <Link
        to="/services/mesures"
        component={(props) => (
          <StyledLink
            onClick={() => props.navigate(props.href)}
            sx={linkStyle}
            display="block"
          >
            <Flex>
              <ArrowBack size="16" />
              <Text
                sx={{ textDecoration: "underline" }}
                fontWeight="bold"
                pl="1"
              >
                {"Retour Ã  la liste"}
              </Text>
            </Flex>
          </StyledLink>
        )}
      />
    </Box>
  );
}

export { ServiceMesureSidebar };
