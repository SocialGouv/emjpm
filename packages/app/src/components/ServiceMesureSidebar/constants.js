export const MESURE_TYPE = {
  CLOSED: "Mesure Ã©teinte",
  IN_PROGRESS: "Mesure en cours",
  WAITING: "Mesure en attente"
};
