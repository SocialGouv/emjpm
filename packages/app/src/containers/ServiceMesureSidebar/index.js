import { ServiceMesureSidebar } from "./ServiceMesureSidebar";

export { ServiceMesureSidebar };
