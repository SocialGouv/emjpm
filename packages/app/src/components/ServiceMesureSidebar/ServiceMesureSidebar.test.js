import React from "react";
import renderer from "react-test-renderer";

import { ServiceMesureSidebar } from "./ServiceMesureSidebar";

it("renders correctly", () => {
  const tree = renderer.create(<ServiceMesureSidebar />).toJSON();
  expect(tree).toMatchSnapshot();
});
