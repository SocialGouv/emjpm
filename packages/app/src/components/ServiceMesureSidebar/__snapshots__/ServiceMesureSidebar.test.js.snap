// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-vurnku"
>
  <a
    className="css-15cod3m"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    <div
      className="css-4cffwv"
    >
      <svg
        aria-hidden="true"
        className="sc-bdVaJa iJCmgd"
        fill="currentColor"
        focusable="false"
        height="16"
        viewBox="0 0 24 24"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 11H6.414l5.293-5.293-1.414-1.414L2.586 12l7.707 7.707 1.414-1.414L6.414 13H21z"
        />
      </svg>
      <div
        className="css-1a8d7sy"
      >
        Retour Ã  la liste
      </div>
    </div>
  </a>
</div>
`;
